wiki/Beatmap_information/en.md

```diff
@@ -0,0 +1,141 @@
+# Beatmap information
+
+The **beatmap information page** displays various statistics about a [beatmap](/wiki/Beatmap), such as its mapper, star rating, and the BPM. This article details the many aspects and features the information page has to offer.
+
+## Difficulty menu
+
+This menu is the main part containing vital information about the beatmap. The background depicts a cutout version of the beatmap's image.
+
+At the top-left, next to the currently selected `info` tab, is the [beatmap discussion](/wiki/Beatmap_discussion) tab used for [modding](/wiki/Modding) a beatmap.
+
+### Mapping information
+
+::: Infobox
+![](img/mapping-info.png "Mapping information")
+:::
+
+::: Infobox
+![](img/game-mode-selection.png "Game mode selection")
+:::
+
+Directly below the tab selection follows the difficulty section presenting every [difficulty](/wiki/Beatmap/Difficulty) in the beatmap. The game mode icons are coloured based on the [star rating colour spectrum](/wiki/Beatmap/Difficulty#difficulty-and-star-rating).
+
+Next to this display is the difficulty's name, the play count, and the number of users that have added this beatmap to their favourites. Hovering over a different icon than the currently selected one reveals the difficulty's name and the star rating. Clicking the icon will select this difficulty as the active one, resulting in any beatmap statistics on the page to change accordingly. When hovering over the favourite count, up to 50 profile icons of users favouriting this beatmap will be displayed.
+
+Difficulties from other [game modes](/wiki/Game_mode) can be accessed by clicking on the respective icon in the game mode selection. The number next to the icons indicates the number of difficulties created by the beatmap host and any guest mappers. Difficulties mapped in [osu!](/wiki/Game_mode/osu!) are automatically available as [converts](/wiki/Beatmap/Converts) in the other game modes.
```

Comment:
Changing from plural to singular in the same sentence. Semantics. Also simplified the second sentence, as it's already implied that will be the total difficulty count? Kinda weird way to say it by mentioning "Host + GD" difficulties.
```suggestion
Difficulties from other [game modes](/wiki/Game_mode) can be accessed by clicking on their respective icons in the game mode selection. The number next to the icons indicates the total number of difficulties for each game mode. Difficulties mapped in [osu!](/wiki/Game_mode/osu!) are automatically available as [converts](/wiki/Beatmap/Converts) in the other game modes.
```